File: 0832-flipping-an-image/0832-flipping-an-image.py
class Solution:
    def flipAndInvertImage(self, image: List[List[int]]) -> List[List[int]]:
        for i in range(len(image)):
            s,e=0,len(image[i])-1
            while(s<=e):
                if(image[i][s]==1 and image[i][e]==1):
                    image[i][s]=0
                    image[i][e]=0
                elif(image[i][s]==0 and image[i][e]==0):
                    image[i][s]=1
                    image[i][e]=1
                s+=1
                e-=1
        return image

File: 0832-flipping-an-image/0832-flipping-an-image.java
class Solution {
    public int[][] flipAndInvertImage(int[][] image) {
        for(int i=0; i<image.length; i++){
            int s=0, e=image[i].length-1;
            while(s<=e){
                if(image[i][s]==1 && image[i][e]==1){
                    image[i][s]=0;
                    image[i][e]=0;
                }else if(image[i][s]==0 && image[i][e]==0){
                    image[i][s]=1;
                    image[i][e]=1;
                }
                s++;
                e--;
            }
        }
        return image;
    }
}